src/number_theory/number_field.lean

```diff
@@ -249,4 +250,83 @@ end
 
 end roots
 
+section bounded
+
+open finite_dimensional polynomial set
+
+variables {K : Type*} [field K] [number_field K]
+variables {A : Type*} [normed_field A] [is_alg_closed A] [normed_algebra ℚ A]
+
+lemma coeff_bdd_of_norm_le {B : ℝ} {x : K} (h : ∀ φ : K →+* A, ∥φ x∥ ≤ B) (i : ℕ):
```

Comment:
I don't remember, did we decide to go with `K →+* A` rather than `ℚ`-algebra morphisms?

Reply:
Yes, we decided that embeddings would be ring homomorphisms.